tests/Collection/LocaleCollectionTest.php

```diff
@@ -70,16 +70,14 @@ public function testGroupByCurrencyFormat()
             ]
         );
 
-        self::assertEquals(
-            [
-                '€ 123.456.789,12' => new LocaleCollection([Locale::fromString('nl_NL')]),
-                '123.456.789,12 €' => new LocaleCollection([Locale::fromString('nl_BE'), Locale::fromString('de_DE')]),
-            ],
-            $collection->groupByCurrencyFormat(new CurrencyIso('EUR'), 123456789.1234)
-        );
+        $result = $collection->groupByCurrencyFormat(new CurrencyIso('EUR'), 123456789.1234);
+
+        self::assertArrayHasKey('€ 123.456.789,12', $result);
+        self::assertArrayHasKey('123.456.789,12 €', $result);
+        self::assertNotEquals($result['€ 123.456.789,12'], $result['123.456.789,12 €']);
```

Comment:
This test lacks verification if the locales with the same formatting are grouped together

Reply:
The issue here is that this is dependent on the ICU version. Thus, you're basically testing ICU here instead of the PHP code, which differs between the 7.2 and 7.4 checks.

Reply:
I even removed this class in https://github.com/MyOnlineStore/common-domain/pull/25 due to the issues described in that PR description 🙂